components/pipeline-service/production/stone-prod-p02/deploy.yaml

```diff
@@ -1203,8 +1341,10 @@ spec:
         app: pipeline-metrics-exporter
     spec:
       containers:
-      - args: []
-        image: quay.io/redhat-appstudio/pipeline-service-exporter:2782659983c94692497467cd1cf952b1bc1f0da4
+      - args:
+        - -pprof-address
+        - "6060"
+        image: quay.io/redhat-appstudio/pipeline-service-exporter:c93b6d93e8bbd4540a4d565a35bae2a8b081b000
```

Comment:
It looks like this cluster was not up to date, thus the change.

Reply:
yep this is OK and lines up with https://github.com/redhat-appstudio/infra-deployments/blob/main/components/pipeline-service/staging/stone-stg-rh01/deploy.yaml#L1349-L1353